clinic/clinic-user-guide-for-tiup.md

```diff
@@ -174,6 +174,7 @@ With Diag, you can collect data from the TiDB clusters and the DM clusters deplo
     - `-N/--node`: only collects data from a specified node. The format is `ip:port`.
     - `--include`: only collects specific types of data. The optional values are `system`, `monitor`, `log`, `config`, and `db_vars`. To include two or more types, you can use `,` as a separator between the types.
     - `--exclude`: does not collect specific types of data. The optional values are `system`, `monitor`, `log`, `config`, and `db_vars`. To exclude two or more types, you can use `,` as a separator between the types.
+    - `--metricsfilter`: Comma separated list of prefixes of Prometheus metrics to collect. For example: `--metricsfilter=tidb,pd` would collect metrics that start with `tidb` and metrics that start with `pd`.
```

Comment:
No as there isn't a fixed list. This is because these are prefixes and not a list of components.

With a TiUP Playground:
![image](https://github.com/user-attachments/assets/4c9d69aa-1aa8-49e6-a3be-fc2e6a449b39)


So `tidb` would match `tidb_server_query_total` but `tidb_server` would also be a valid prefix.

Reply:
Some idea about how many metrics there are and what prefixes could be used:
```
$ curl -s 'http://127.0.0.1:9090/api/v1/label/__name__/values' | jq -r '.data[]' | cut -d\_ -f1 | uniq -c | sort -rn
    446 tikv
    365 tidb
    178 prometheus
    142 pd
    112 etcd
    107 go
     40 raft
     24 resource
      7 process
      7 grpc
      6 net
      4 scrape
      3 br
      2 promhttp
      2 os
      1 up
      1 tso
      1 service
```